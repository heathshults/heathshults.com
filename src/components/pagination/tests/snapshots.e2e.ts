import { snapIt } from '../../../test';

const component = 'pagination';

describe(component, () => {
  const snap = snapIt(component);

  describe('renders', () => {
    snap('should work without parameters', '<hs-pagination></hs-pagination>');
    snap('renders the correct buttons when pages set', '<hs-pagination pages="15"></hs-pagination>');
  });
});
