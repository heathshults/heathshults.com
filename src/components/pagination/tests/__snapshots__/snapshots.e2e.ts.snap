// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pagination renders renders the correct buttons when pages set 1`] = `"<hs-pagination pages=\\"15\\" class=\\"hydrated\\"><nav class=\\"hs-pagination\\"><button class=\\"hs-pagination__control\\" disabled=\\"\\">‹</button><div class=\\"hs-pagination__pages\\"><button class=\\"hs-pagination__control\\" aria-current=\\"\\">1</button><button class=\\"hs-pagination__control\\">2</button></div><button class=\\"hs-pagination__control\\">›</button></nav></hs-pagination>"`;

exports[`pagination renders should work without parameters 1`] = `"<hs-pagination class=\\"hydrated\\"><nav class=\\"hs-pagination\\"><button class=\\"hs-pagination__control\\" disabled=\\"\\">‹</button><div class=\\"hs-pagination__pages\\"><button class=\\"hs-pagination__control\\" aria-current=\\"\\">1</button></div><button class=\\"hs-pagination__control\\" disabled=\\"\\">›</button></nav></hs-pagination>"`;
